src/Events/Custom_Tables/V1/WP_Query/Monitors/WP_Query_Monitor.php

```diff
@@ -26,20 +25,6 @@
 class WP_Query_Monitor implements Countable {
 	use Query_Monitor;
 
-	/**
-	 * A list of possible modifiers implementations.
-	 *
-	 * @since 6.0.0
-	 *
-	 * @var array<string>
-	 */
-	private $implementations = [
```

Comment:
I would not remove this property from the class, just remove the ECP entries. See my comment in the trait.